parent/pom.xml

```diff
@@ -692,6 +692,12 @@
                 <artifactId>athenz-zts-java-client</artifactId>
                 <version>${athenz.version}</version>
             </dependency>
+            <dependency>
+                <groupId>com.github.tomakehurst</groupId>
+                <artifactId>wiremock-standalone</artifactId>
+                <version>2.6.0</version>
+                <scope>test</scope>
```

Comment:
Please never use scope in dependencyManagement.
https://stackoverflow.com/questions/15221299/dependencymanagement-and-scope
https://stackoverflow.com/questions/47394837/is-it-good-practice-to-define-scope-of-dependencies-in-maven-bom-bill-of-materi/47402262#47402262